### ISSUE TITLE
New Version (0.39) of websocket client is not compatible with library

### ISSUE THREAD

**Issue by wilsonge** [feus4177/socketIO-client-2#5]
Error:

```
An incompatible websocket library is conflicting with the one we need.
You can remove the incompatible library and install the correct one
by running the following commands:

yes | pip uninstall websocket websocket-client
pip install -U websocket-client
```

Regressing back to the 0.37 version works fine

The message comes from https://github.com/feus4177/socketIO-client-2/blob/master/socketIO_client/transports.py#L17 

However running this line with the import from CLI works just fine. Unfortunately didn't have time to debug further :(

**Comment by wilsonge** [comment]
Looks like you need to pull in https://github.com/invisibleroads/socketIO-client/pull/139

**Comment by feus4177** [comment]
I looked at websocket-client 0.39 and 0.40 and they both have the `create_connection` attr so I don't see why you would be getting that error.

**Comment by feus4177** [comment]
I made a fix for the SSLError import issue though: eb2de1cca15c7c71282cfe0729c15225ff797184

**Comment by wilsonge** [comment]
Seems good 👍
